v1.1/uuid.md

```diff
@@ -0,0 +1,135 @@
+
+## Size
+A `UUID` column is 128 bits in width, but the total storage size is likely to be larger due to CockroachDB metadata. 
+
+## Examples
+
+### Create a table with manually-entered UUID values
+
+#### Create a table with UUID in standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format.
+
+~~~ sql
+> CREATE TABLE v (token uuid);
+
+> INSERT INTO v VALUES ('63616665-6630-3064-6465-616462656562');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
++--------------------------------------+
+(1 row)
+~~~
+
+#### Create a table with UUID in standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format with braces.
+
+~~~ sql
+> INSERT INTO v VALUES ('{63616665-6630-3064-6465-616462656563}');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
+| 63616665-6630-3064-6465-616462656563 |
++--------------------------------------+
+(2 rows)
+~~~
+
+#### Create a table with UUID in byte format.
+
+~~~ sql
+> INSERT INTO v VALUES (b'kafef00ddeadbeed');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
+| 63616665-6630-3064-6465-616462656563 |
+| 6b616665-6630-3064-6465-616462656564 |
++--------------------------------------+
+(3 rows)
+~~~
+
+#### Create a table with UUID used as URN.
+
+~~~ sql
+> INSERT INTO v VALUES ('urn:uuid:63616665-6630-3064-6465-616462656564');
+
+> SELECT * FROM v;
+~~~
+
+~~~
++--------------------------------------+
+|                token                 |
++--------------------------------------+
+| 63616665-6630-3064-6465-616462656562 |
+| 63616665-6630-3064-6465-616462656563 |
+| 6b616665-6630-3064-6465-616462656564 |
+| 63616665-6630-3064-6465-616462656564 |
++--------------------------------------+
+(4 rows)
+~~~
+
+### Create a table with auto-generated unique row IDs
+
+To auto-generate unique row IDs, you can cast UUID as bytes and use the uuid_v4() function. 
+
+~~~ sql
+> CREATE TABLE w (id UUID PRIMARY KEY DEFAULT uuid_v4()::UUID, name STRING);
+
+> SELECT * from w;
+~~~
+
+~~~
++--------------------------------------+------+
+|                  id                  | name |
++--------------------------------------+------+
+| 7f9c24e8-3b12-4fef-91e0-56a2d5a246ec | Nuna |
++--------------------------------------+------+
+(1 row)
+~~~
+
+{{site.data.alerts.callout_info}}When using UUID with uuid_v4() as the default value, consecutively generated IDs will be spread across different key-value ranges (and therefore likely across different nodes).{{site.data.alerts.end}}
```

Comment:
Need to html code-format a few things:
- `<code>UUID</code>`
- `<code>uuid_v4()</code>`